README.md

```diff
@@ -1513,4 +1513,10 @@ And meet my cat who saw some things in this world :D
 
 ![Cat](assets/images/frustrating-tisha.jpg)
 
+---
+## Hello, Kottans!
```

Comment:
Hey Maryna!

Great job! 👍 

I see that you try to use different markdown content (headers, bold etc.). And if you want to apply them here you have to add a new line between content sections.

Reply:
UPD: you can use different tools to preview you markdown in realtime (for example, for VSCode `shift+cmd+v` (`ctrl+v` for Windows).
Please, feel free to ping me you if you need more details.

Reply:
Thanx! I added a new line between content sections. Is it correct? )

Reply:
Hello, should I do something more? I added a new line. Is it correct? )

Reply:
Great!
Looks good to me!